internal/commands/test.go

```diff
@@ -169,6 +192,14 @@ func checkoutNewBranch(tempBranchName string, tmpDir string) error {
 }
 
 func commitChanges(tmpDir string, commitMessage string) (string, error) {
+	if v, ok := os.LookupEnv("SENTRY_DEBUG_FAIL"); ok && strings.HasPrefix(v, "commitChanges") {
+		if _, r, _ := strings.Cut(v, "="); r == "panic" {
```

Comment:
@nikandfor I'm assuming this block allows testing the Sentry integration - what does the `=` symbol mean here? Maybe a comment that mentions the intended usage would help?